doc/mgr/orchestrator.rst

```diff
@@ -425,25 +499,31 @@ to specify the deployment of services.  For example:
         - host1
         - host2
         - host3
-    spec: ...
```

Comment:
to make things a tiny bit more obvious:

`ceph orch ls ` calls:

https://github.com/ceph/ceph/blob/fa53f1032fc2e3bdce437797a09676dff58426a3/src/pybind/mgr/cephadm/module.py#L1527-L1529

Which returns a `CephadmCompletion[List[orchestrator.ServiceDescription]]`

`describe_service` is called by 

https://github.com/ceph/ceph/blob/fa53f1032fc2e3bdce437797a09676dff58426a3/src/pybind/mgr/orchestrator/module.py#L491-L496

Then we're running

https://github.com/ceph/ceph/blob/fa53f1032fc2e3bdce437797a09676dff58426a3/src/pybind/mgr/orchestrator/module.py#L491-L496

where `data` is of type `List[ServiceSpec]`

thus `ceph orch ls --export` returns the ServiceSpecs instead of CephadmDaemonSpecs.

helps?